packages/rust/build.sh

```diff
@@ -92,7 +96,9 @@ termux_step_make_install() {
 }
```

Comment:
just above this (line 94) there's still a reference to libLLVM-10.0.1.so
i think we need an _LLVM_VER variable

Reply:
Added it.